src/frontend/src/req_convert/insert/column_to_row.rs

```diff
@@ -35,6 +35,6 @@ fn request_column_to_row(request: InsertRequest) -> Result<RowInsertRequest> {
     Ok(RowInsertRequest {
         table_name: request.table_name,
         rows: Some(rows),
-        region_number: request.region_number,
+        region_number: 0, // deprecated field
```

Comment:
Adds a `FIXME` label.

Reply:
Or maybe we can remove it immediately?

Reply:
https://github.com/GreptimeTeam/greptime-proto/pull/101